src/transformers/commands/add_new_model_like.py

```diff
@@ -173,6 +173,63 @@ def parse_module_content(content: str) -> List[str]:
     return objects
 
 
+def extract_blocks(content: str, indent_level: int = 0) -> List[str]:
+    """Parse the blocks in `content` into the list of objects it defines.
+
```

Comment:
Basically just `parse_module_content` but could deal with indent level.

Also removed the logic below
```
        if is_valid_object and len(current_object) == 1:
            is_valid_object = not current_object[0].startswith("# Copied from")
```